build.psm1

```diff
@@ -307,7 +307,8 @@ function Start-PSBuild {
         [ValidateNotNullOrEmpty()]
         [string]$ReleaseTag,
         [switch]$Detailed,
-        [switch]$InteractiveAuth
+        [switch]$InteractiveAuth,
+        [switch]$SkipAnalyzers
```

Comment:
Maybe:
```suggestion
        [switch]$SkipCSharpAnalyzers
```
I think about PSSA :-)

Reply:
Not all analyzers are C#, e.g.: [IL3000](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/il3000).

I prefer `SkipRoslynAnalyzers` in this case.